ui/Playlists.php

```diff
@@ -1513,9 +1534,10 @@ public function emitImportJSON() {
                     $playlist = Engine::lastInsertId();
 
                     // insert the tracks
+                    $status = '';
                     foreach($json->data as $pentry) {
                         $entry = PlaylistEntry::fromJSON($pentry);
-                        $success = $papi->insertTrackEntry($playlist, $entry, false);
+                        $success = $papi->insertTrackEntry($playlist, $entry, null, $status);
                         // if there is a timestamp, set it via update
                         if($success && $pentry->created)
                             $papi->updateTrackEntry($playlist, $entry);
```

Comment:
Now that you're letting insertTrackEntry set the timestamp, you can remove this call to updateTrackEntry.  It was necessary previously, because insertTrackEntry did not set the timestamp in this situation.

Reply:
done.